### DISCUSSION TITLE
high versions jumps in the signature version

### DISCUSSION THREAD

**Discussion by OFfriend** [UltimaHoarder/UltimaScraper#1436] [Show and tell]
Onlyfans has rebuilt the website after version `77`. With minimal adjustments my tool is running again. Since then I always see big version jumps! For example: `77` -> `85` & `87` -> `94` Maybe the developers are no longer testing in production. 😂

**Comment by UltimaHoarder** [comment]
I still always laugh at how they tried to stop people from accessing the dev tools by introducing infinite loops which slowed down the site 😂

**Reply by OFfriend** [reply]
True right click -> `Never pause here` -> Done 😂
